bugsnag-android-core/src/test/java/com/bugsnag/android/MetadataRedactionTest.java

```diff
@@ -0,0 +1,68 @@
+package com.bugsnag.android;
+
+import static com.bugsnag.android.JsonUtilsKt.validateJson;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MetadataRedactionTest {
+
```

Comment:
Should we include a test for "password" being the default redacted key?

Reply:
This is covered in a [separate test](https://github.com/bugsnag/bugsnag-android/blob/json-redaction/bugsnag-android-core/src/androidTest/java/com/bugsnag/android/ManifestConfigLoaderTest.kt#L49) which verifies 'password' is the default value of `redactKeys`

Reply:
This would be the default even if Manifest files were not used. This feels like a lower-level requirement that should have a test here unless you are going to handle the default value in Configuration and not have any default at this level.

Reply:
Added an additional test case to verify the default value on `Metadata` specifically.